app/src/main/java/com/firebase/uidemo/auth/AuthUiActivity.java

```diff
@@ -155,17 +156,25 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean checked) {
 
     @OnClick(R.id.sign_in)
     public void signIn(View view) {
-        startActivityForResult(
-                AuthUI.getInstance().createSignInIntentBuilder()
-                        .setTheme(getSelectedTheme())
-                        .setLogo(getSelectedLogo())
-                        .setAvailableProviders(getSelectedProviders())
-                        .setTosAndPrivacyPolicyUrls(getSelectedTosUrl(),
-                                getSelectedPrivacyPolicyUrl())
-                        .setIsSmartLockEnabled(mEnableCredentialSelector.isChecked(),
-                                mEnableHintSelector.isChecked())
-                        .build(),
-                RC_SIGN_IN);
+        FirebaseAuth.getInstance()
```

Comment:
If you look at my (now-closed) PR I added a whole separate sample `Activity` for anonymous.  You might as well copy it, IMO it shows off the flow nicely without complicating the "normal" one.

Reply:
Done.